### PR TITLE
added variable play back speed, fixed the video bar

### PR DESCRIPTION
**Pull request by aryanbisht-dev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "react": "^18.2.0",
     "react-dom": "^18.2.0",
     "react-scripts": "5.0.1",
-    "web-vitals": "^2.1.0"
+    "web-vitals": "^2.1.0",
+    "yarn": "^1.22.19"
```
Comment (AntiAnt) [inline package.json:13]: did we add yarn locally or globally?

File: src/App.js [hunk 2]
```diff
@@ -111,6 +110,26 @@ function App() {
...
+      case "4x":
+        video.current.playbackRate = 4;
+        break;
+      case ".5x":
+        video.current.playbackRate = .5;
+        break;
+      case ".25x":
+        video.current.playbackRate = .25;
+        break;
+    }
```
Comment (AntiAnt) [inline src/App.js:132]: Should we have a default case?

**Review by AntiAnt — CHANGES_REQUESTED** [review]
good we can go over the comments after lunch

File: src/App.js (hunk 2 above)
Comment (aryanbisht-dev) [reply src/App.js:132]: Yes, I am missing a default

**Review by AntiAnt — APPROVED** [review]

**Comment by aryanbisht-dev** [comment]
Merging playback speed and normalized bar

**Pull request merged**